### PR TITLE
v1.0.6

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/my-share-card/MyShareCard.tsx [hunk 1]
```diff
@@ -28,6 +28,21 @@ function MyShareCard({
   if (!data) return null
   const { type: poolType } = POOLS_MAP[data.name]
   const formattedDecimals = poolType === PoolTypes.USD ? 2 : 4
+  const hasAVAX = 
+    data.masterchefBalance?.pendingTokens.bonusTokenAddress.toLowerCase()
```
Comment (prodvdc) [inline src/components/my-share-card/MyShareCard.tsx]: Do we need to have optional chain operator here? If needed, we might need to have optional chains as following.
data.masterchefBalance?.pendingTokens?.bonusTokenAddress?.toLowerCase()

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-develop.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-develop.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-develop.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-develop.dev-app.axial.exchange

File: src/components/my-share-card/MyShareCard.tsx (hunk 1 above)
Comment (saito-sv) [reply src/components/my-share-card/MyShareCard.tsx]: is this still needs to be handled?

Comment (ghost) [reply src/components/my-share-card/MyShareCard.tsx]: fix at #66

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-develop.dev-app.axial.exchange

**Review by prodvdc — APPROVED** [review]
LGTM. Let's go live!

**Pull request merged**